src/xpcc/architecture/platform/board/nucleo_l476rg/nucleo_l476rg.cpp

```diff
@@ -8,6 +8,7 @@
 
 // Create an IODeviceWrapper around the Uart Peripheral we want to use
 Board::LoggerDevice loggerDevice;
+xpcc::IOStream stream(loggerDevice);
```

Comment:
Do we want to expose this at a board level? Maybe move to the `main.cpp` of the example. In general debugging/tracing could be improved in xpcc, but it's a hard problem, and Uart works reasonably okish.

Reply:
> move to the main.cpp of the example

Must be globally available by the driver. I do not like `extern` or whatever.
Maybe it is just left to the user to add the `stream` when necessary for debugging. Debugging I2C is not for the faint-hearted, btw.